trade-finance-manager-api/src/v1/mappings/map-submitted-deal/map-cash-contingent-facility.js

```diff
@@ -58,6 +59,7 @@ const mapCashContingentFacility = (facility) => {
     feeType: mapGefFacilityFeeType(paymentType),
     feeFrequency,
     dayCountBasis,
+    hasBeenIssuedAndAcknowledged,
```

Comment:
Need to update tests for changes to functions in `map-submitted-deal`

Reply:
done